doc/source/how-to/packaging.rst

```diff
@@ -334,23 +337,61 @@ The ``dependabot.yml`` file
 
 Dependabot version updates are performed by checking a ``dependabot.yml``
 configuration file into your repository. In this file, one should specify the
-location of the project's requirement files, so that Dependabot knows where
-to look.
+location of the project's requirement files, so that Dependabot knows where to
+look. On top of that, Dependabot is also capable of updating ``GitHub actions``
+versions.
```

Comment:
Does GitHub actions need to be a literal string. It seems a common term and not something that is code-specific. Also, I'd recommend "...capable of updating versions for GitHub actions."

Reply:
You are right Kathy. No need to have this sort of highlight on the GitHub actions word.